integration-test.sh

```diff
@@ -18,6 +18,8 @@ set -ex
 GCS_BUCKET="${GCS_BUCKET:-gs://kaniko-test-bucket}"
 IMAGE_REPO="${IMAGE_REPO:-gcr.io/kaniko-test}"
 
+docker version
```

Comment:
just 1 nit: you can remove this here and log it in the newly added  `getDockerMajorVersion`

Reply:
getDockerMajorVersion uses a template string to only get engine version. The command in script shows more info that's useful for debugging so I think should be kept as is.